setup.py

```diff
@@ -19,7 +19,7 @@
 
 setuptools.setup(
     name="verticapy",
-    version="0.9.0",
+    version="0.9.0-0",
```

Comment:
Let's keep the version numbers simple and only 3 digits: so `0.9.1` instead of `0.9.0-0`.

Reply:
I know but we will not release it. It is mainly to avoid collision when a user is pulling the repo with the version 0.9.1 (not the real one but just the last merge), he will not be able to upgrade to 0.9.1 (when this version is released). This is why I am adding an extra digit.

Reply:
Did we have such a problem before? Are you planning to increase the forth digit on every PR from now on?

Reply:
Yes we had. No, I am adding this digit only for the PR which are not releases